test/functional/cmdLineTests/xlpCMLTests/playlist.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright (c) 2018, 2020 IBM Corp. and others
```

Comment:
This is a new file, so it should be 2020, 2020

Reply:
Updated to 2020, 2020. Thanks.